src/test/ui/extern-flag/nounused.rs

```diff
@@ -0,0 +1,7 @@
+// check-pass
```

Comment:
you may be able to use revisions to consolidate this into one test, but I guess I won't block this on that.

Reply:
What do you mean?